superbench/benchmarks/micro_benchmarks/micro_base.py

```diff
@@ -181,13 +182,13 @@ def _benchmark(self):
                         self._curr_run_index, self._name, output.stdout
                     )
                 )
-                return False
+                success = False
             else:
                 if not self._process_raw_result(cmd_idx, output.stdout):
                     self._result.set_return_code(ReturnCode.MICROBENCHMARK_RESULT_PARSING_FAILURE)
-                    return False
+                    success = False
 
-        return True
+        return success
```

Comment:
It looks like we don't have return code for each?

Reply:
yes